specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AutoProvisioningAwsAccount/CreateAutoProvisioningSettingsSubscription_example.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "properties": {
+      "awsAccessKeyId": "AKIARPZCNODDJSMKFX4J",
+      "awsSecretAccessKey": "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"
```

Comment:
model validation says you are missing a parameter of `AutoProvisioningAwsAccountCredentials` here, if I did not get it wrong.